src/osd/OSD.cc

```diff
@@ -2292,6 +2292,19 @@ will start to track new ops received afterwards.";
     f->open_object_section("compact_result");
     f->dump_float("elapsed_time", duration);
     f->close_section();
+  } else if (admin_command == "get_mapped_pools") {
+    f->open_object_section("mapped_pools");
+    set<int> poollist = get_mapped_pools();
+
+    std::stringstream buffer;
+    for (set<int>::iterator it = poollist.begin();
```

Comment:
please see `types.h`, we have helpers for this purpose in our arsenal.

Reply:
addressed, thanks